.github/workflows/daily-integration-check.yml

```diff
@@ -34,3 +34,24 @@ jobs:
         with:
           filename: .github/integration-check-failure-issue.md
           update_existing: true
+
+  load-testing-check:
+    name: Load testing check
+    env:
+      SUPERSTAQ_API_KEY: ${{ secrets.SUPERSTAQ_API_KEY }}
+      TEST_USER_IBMQ_TOKEN: ${{ secrets.TEST_USER_IBMQ_TOKEN }}
```

Comment:
nit: can we move the `env` map outside the `jobs` section since both jobs use it? not sure if yaml syntax allows this...